src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -168,6 +176,69 @@ private IEnumerable<CompletionItem> GetResourceTypeCompletions(SemanticModel mod
                 .ToList();
         }
 
+        private IEnumerable<CompletionItem> GetModulePathCompletions(SemanticModel model, BicepCompletionContext context)
+        {
+            if (!context.Kind.HasFlag(BicepCompletionContextKind.ModulePath)
+            || FileResolver.TryResolveModulePath(model.SyntaxTree.FileUri, ".")?.LocalPath is not string cwd
```

Comment:
Using `LocalPath` feels like you're letting the filesystem abstraction slip through. I think you should be able to just use `model.SyntaxTree.FileUri` instead of cwdUri in FileResolver.TryResolveModulePath.